src/System.Net.Libuv/src/project.json

```diff
@@ -1,12 +1,14 @@
-{
+﻿{
   "dependencies": {
-    "System.Diagnostics.Debug": "4.0.0", 
+    "System.Buffers": "0.1.0-d103015-1",
```

Comment:
Where did this weird version number came from? :) I think you can add "version" : "1.2.3.4" in the System.Buffers to change it